kbase-extension/static/kbase/config/feature-config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "stagingDataViewer": false,
+    "stagingDataViewer": true,
```

Comment:
We should probably manage this file using a Docker volume mount in each environment anyway.  So I'm merging.